tests/unit/test_convert.py

```diff
@@ -118,15 +91,15 @@ def test_copy_attrs(self):
                 dataset.setncattr('attr_three', 'val_three')
                 dataset.setncattr('attr_two', 'val_four')
 
-                copy_attrs(dataset, zarr_group, {'scaled': 1.0})
+                copy_attrs(dataset, zarr_group, kwarg_attr=1.0)
```

Comment:
```suggestion
                copy_attrs(dataset, zarr_group, kwarg_attr=1.0, attr_one='unchanged')
```
Just to also show that you can't even overwrite an attribute even if you explicitly add it to the kwargs.

Reply:
Nice. I'll add that.